node/resolvers/profile/index.ts

```diff
@@ -111,41 +111,28 @@ export const mutations = {
       }).catch(returnOldOnNotChanged(oldData))
   },
 
-  updateProfilePicture: async (root, { file, field }, ctx, info) => {
+  updateProfilePicture: async (root, args, ctx, info) => {
+    const file = args.file
```

Comment:
I think you can do `const { file, field || 'profilePicture' } = args `

Reply:
Does not work :(